.docker/php/Dockerfile

```diff
@@ -10,11 +10,11 @@ COPY --from=composer /usr/bin/composer /usr/bin/composer
 
 RUN apt-get update \
```

Comment:
```suggestion
# hadolint ignore=DL3008
RUN apt-get update \
```

**suggestion:** adding this line will allow unpinned package version : https://github.com/hadolint/hadolint/wiki/DL3008

Reply:
@arnaud-deabreu Thanks for this info. Do you know, if there is a mode to use the pinned versions, with a fallback to unpinned versions?

If not, i guess it is the best, that i revert my changes. And only add your suggested commit (with a comment) to this PR.

Reply:
I think you could try with something like this:
```shell
apt-get satisfy -qq --yes --no-install-recommends \
    "git (>= 1:2.30.2), git (<< 1:3)"
```